packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/local.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import LocalFooter from './local-footer';
+import WorkspaceControl from './workspace-control';
+import RootStore from 'Stores/index';
+import './index.scss';
+
+type Nullable<T> = T | null;
+type TLocalComponent = {
+    handleFileChange: (e: React.ChangeEvent<HTMLInputElement>, data: boolean) => boolean;
+    is_mobile: boolean;
+    loaded_local_file: boolean;
+    setLoadedLocalFile: (data: Nullable<string>) => void;
+    onConfirmSave: () => void;
+    onDrop: () => void;
+    setActiveTab: (param: number) => void;
+    loadFileFromLocal: () => void;
+    previewRecentStrategy: () => void;
+    loadFileFromRecent: () => void;
+    setFileLoaded: (param: boolean) => void;
+};
+
+const LocalComponent = ({
+    handleFileChange,
+    is_mobile,
+    loaded_local_file,
+    setLoadedLocalFile,
+    onConfirmSave,
+    setActiveTab,
+    loadFileFromLocal,
+    loadFileFromRecent,
+    setFileLoaded,
+}: TLocalComponent) => {
+    const file_input_ref = React.useRef<HTMLInputElement | null>(null);
+    const clear_preview_ref = React.useRef<HTMLInputElement | null>(null);
+    const [is_file_supported, setIsFileSupported] = React.useState<boolean>(true);
+    const loadedLocalFileLocation = () => {
+        return loaded_local_file ? loadFileFromLocal() : loadFileFromRecent();
+    };
+    const clearInjectionDiv = () => {
+        const element = document.getElementById('load-strategy__blockly-container');
+        if (element?.getElementsByClassName('injectionDiv').length) {
+            element.removeChild(element.getElementsByClassName('injectionDiv')[0]);
+        }
+    };
```

Comment:
Tried to use `Blockly.mainWorkspace.clear()` method but it does not remove the injection DIV.
For now removing the injection div with javascript need to find a workaround with blockly.